package.json

```diff
@@ -21,8 +21,10 @@
   "main": "./lib/node-pre-gyp.js",
   "dependencies": {
     "detect-libc": "^1.0.3",
+    "http-proxy-agent": "^4.0.1",
+    "https-proxy-agent": "^5.0.0",
```

Comment:
One of the reasons I recall you mentioning node-fetch as better than needle was size. After adding these extra deps, does it still look good for fetch overall in terms of being nice and small? e.g. are either of these quite big?